### DISCUSSION TITLE
`sea-orm-cli` overwrites code when re-generating entity files.

### DISCUSSION THREAD

**Discussion by aspiringLich** [SeaQL/sea-orm#2142] [Q&A]
After writing a relation for one of the generated entities:

```rs
#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
#[sea_orm(table_name = "google_user")]
pub struct Model {
    #[sea_orm(unique)]
    pub user_id: Uuid,
    #[sea_orm(primary_key, auto_increment = false)]
    pub google_id: String,
}

#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {
    #[sea_orm(
        belongs_to = "super::user::Entity",
        from = "Column::UserId",
        to = "super::user::Column::UserId"
    )]
    User,
}

impl Related<super::user::Entity> for Entity {
    fn to() -> RelationDef {
        Relation::User.def()
    }
}
```

When i run `sea-orm-cli generate entity` again, it overwrites the `Relation` and `Related` definition. Since the documentation directs you write these by hand, why are they overwritten every time? I'm almost certainly missing something, or going about this wrong, so any help would be appreciated.

**Comment by emmiegit** [comment]
I certainly understand overwriting being an expected behavior (for instance, it lets you do a diff against the checked-in version), but it would be helpful if there was a flag you could specify, say in the generated file you have edited, that tells `sea-orm-cli` that it should not overwrite this file because you have made manual tweaks.
